cards/README.md

```diff
@@ -1,84 +1,119 @@
 
-#### API Playground
-To test adding cards to a program, go to the [**API Playground**](https://dashboard.fidel.uk/playground), and choose **Add Card** from the left menu endpoints.  The method is set to POST and the endpoint to **_/cards_**.  When you choose the program that you would like to connect the card to from the dropdown, the POST url will change to show the program ID.  In the edit field of the playground is the JSON object that will be used to add the card. To add a card, use one of the available testing card numbers displayed below, an expiry date in the future, and the three letter `countryCode`. You must set `termsOfUse` to `true` to define that the user agreed to the terms of use and opt-in.
+### SDKs
+
+We recommend you use the secure and PCI compliant [Web SDK](/web-sdk) or [Mobile SDKs](/mobile-sdks) to create Cards in the Fidel environments. The SDKs work in both the live and test environment and use your SDK key. You can find your SDK keys (Live and Test) in your [Dashboard Account Settings](https://dashboard.fidel.uk/account/plan).
+
+All our SDKs require a user to enter their card number and expiry date, along with the country of issue for the card. The SDKs won't require the CVV number, and will not make any active card checks against the cards. The SDKs will pre-populate the `countryCode` and the `programId` of the Program you want to link the card to. The card numbers are tokenised and transmitted directly from our secure pre-built SDKs to our API on submission. This way, your servers are never exposed to sensitive information, removing all PCI compliance requirements for you.
 
 <div class="info-box">
-    <small>Important note</small><br/>
-    To use the <strong>Create Card</strong> endpoint, you must use the test public key. Using the <strong>Create Card</strong> API endpoint on live environment requires your company to be PCI Compliant. If you want to use the API instead of the SDKs, please contact us at devrel@fidel.uk.
+  <small>Linking Multiple Cards</small><br/>
+  To facilitate users linking multiple cards, add identifying key:value pairs from your system in the metadata field. You can read more in the [Metadata](/cards#Metadata) section below.
 </div>
 
-![Create card](https://raw.githubusercontent.com/FidelLimited/docs/master/assets/images/create-card.png "Create card")
-
-When the card is successfully linked, the newly created card object is returned in JSON and displayed in the response body box. If the card linking fails, the failure object will be delivered instead.  
+### API
+
+If you don't want to use our secure and PCI compliant SDKs, you must get PCI Compliant before using our Cards API. Contact us at devrel@fidel.uk for more information.
+
+```sh
+curl -X POST \
+  https://api.fidel.uk/v1/programs/f76ed1be-e434-480b-aa1d-ff48f548f62a/cards
+  -H 'Content-Type: application/json'
+  -H 'Fidel-Key: pk_test_62f02030-0409-4eb5-ab94-6eff05b3d888'
+  -d '{
+  	"number": "4444000000004222",
```

Comment:
This looks a little bit too on the left, not sure if it will be showed or not on the page

Reply:
<img width="839" alt="Screenshot 2021-02-08 at 11 38 37" src="https://user-images.githubusercontent.com/690132/107214771-4da3ee80-6a02-11eb-854d-659ff58b5457.png">

Reply:
That's how it looks on the preview. Is that ok @danitre ?

Reply:
Yep, all good!